### PR TITLE
Add missing german tanslations

### PR DESCRIPTION
**Pull request by elkuku**
Some missing translations.

Do you have any kind of translation policies and/or online translation tools currently enabled for this project? (e.g. Transifex, Crowdin, etc.)

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Thanks for contributing these translations. I'll wait for the review of our German-speaking friends. About your question: we don't have any policy about this or use any of the tools you mentioned. Thanks!

File: app/Resources/translations/messages.de.xlf [hunk 1]
```diff
@@ -276,6 +276,30 @@
                 <source>help.more_information</source>
                 <target><![CDATA[Für mehr Informationen schaue in die offizielle <a href="http://symfony.com/doc">Symfony Dokumentation</a>.]]></target>
             </trans-unit>
+            <trans-unit id="action.show_post">
+                <source>action.show_post</source>
+                <target>Beitrag anzeigen</target>
+            </trans-unit>
+            <trans-unit id="rss.title">
+                <source>rss.title</source>
+                <target>Symfony Demo blog</target>
```
Comment (xabbuh) [inline app/Resources/translations/messages.de.xlf]: Blog

File: app/Resources/translations/messages.de.xlf [hunk 2]
```diff
@@ -276,6 +276,30 @@
...
+            </trans-unit>
+            <trans-unit id="menu.choose_language">
+                <source>menu.choose_language</source>
+                <target>Sprache wählen</target>
+            </trans-unit>
+            <trans-unit id="rss.description">
+                <source>rss.description</source>
+                <target>Die neusten Posts die auf dem Symfony Demo Blog veröffentlicht wurden</target>
```
Comment (xabbuh) [inline app/Resources/translations/messages.de.xlf]: Die neuesten Posts, die auf dem Symfony Blog veröffentlicht wurden.

**Comment by elkuku** [comment]
Fixed :wink:

**Comment by elkuku** [comment]
The security check is failing on my language file changes - I love Travis :tongue:

**Comment by elkuku** [comment]
While playing wit #531 I discovered more missing strings... @xabbuh would you mind to re-check? :wink:

File: app/Resources/translations/messages.de.xlf [hunk 3]
```diff
@@ -276,6 +276,78 @@
...
+                <source>label.tags</source>
+                <target>Tags</target>
+            </trans-unit>
+            <trans-unit id="http_error.name">
+                <source>http_error.name</source>
+                <target>Fehler %status_code%</target>
+            </trans-unit>
+            <trans-unit id="http_error.description">
+                <source>http_error.description</source>
+                <target>Ein unbekannter Fehler (HTTP %status_code%) ist aufgetreten wegen dem ihre Anfrage nicht ausgeführt werden konnte.</target>
```
Comment (xabbuh) [inline app/Resources/translations/messages.de.xlf]: I would reword this:

> Ein unbekannter Fehler (HTTP %status_code%) ist aufgetreten. Ihre Anfrage konnte deswegen nicht ausgeführt werden.

Comment (xabbuh) [inline app/Resources/translations/messages.de.xlf]: By the way, it looks like everywhere else we used "du" instead of "Sie".

File: app/Resources/translations/messages.de.xlf [hunk 4]
```diff
@@ -276,6 +276,78 @@
...
+                <source>http_error_403.suggestion</source>
+                <target>Fragen Sie Ihren Manager oder Systemadministrator, um Ihnen Zugriff auf diese Ressource zu gewähren.</target>
+            </trans-unit>
+            <trans-unit id="http_error_404.description">
+                <source>http_error_404.description</source>
+                <target>Wir konnten die von Ihnen angeforderte Seite nicht finden.</target>
+            </trans-unit>
+            <trans-unit id="http_error_404.suggestion">
+                <source>http_error_404.suggestion</source>
+                <target>Überprüfen Sie Rechtschreibfehler in der URL or <a href="%url%">gehen sie zurück zur Homepage</a>.</target>
```
Comment (xabbuh) [inline app/Resources/translations/messages.de.xlf]: or -> oder

File: app/Resources/translations/messages.de.xlf [hunk 5]
```diff
@@ -276,6 +276,78 @@
...
+            </trans-unit>
+            <trans-unit id="http_error_500.description">
+                <source>http_error_500.description</source>
+                <target>Es gab einen internen Serverfehler.</target>
+            </trans-unit>
+            <trans-unit id="http_error_500.suggestion">
+                <source>http_error_500.suggestion</source>
+                <target>Versuchen sie diese Seite in einigen Minuten erneut zu laden or <a href="%url%">gehen sie zurück zur Homepage</a>.</target>
```
Comment (xabbuh) [inline app/Resources/translations/messages.de.xlf]: or -> oder

File: app/Resources/translations/messages.de.xlf [hunk 6]
```diff
@@ -276,6 +276,78 @@
...
+                <source>title.login_help</source>
+                <target>Versuchen Sie einen der folgenden Benutzer</target>
+            </trans-unit>
+            <trans-unit id="notification.comment_created">
+                <source>notification.comment_created</source>
+                <target>Ihr Beitrag hat einen Kommentar erhalten!</target>
+            </trans-unit>
+            <trans-unit id="notification.comment_created.description">
+                <source>notification.comment_created.description</source>
+                <target>Ihr Beitrag "%title%" hat einen Kommentar erhalten. Sie können den Kommentar lesen venn sie <a href="%link%">diesem Link</a> folgen.</target>
```
Comment (xabbuh) [inline app/Resources/translations/messages.de.xlf]: [...] lesen, wenn [...]

**Comment by elkuku** [comment]
Fixed, Thanks!

I also changed everything to "informal".

BTW: @xabbuh do you think #531 makes any sense? How do *you* check for missing translations? Don't you think using an online translation service (Transifex, Crowdin, etc.) would help?

Thanks again :wink:

**Comment by javiereguiluz** [comment]
Thank you @elkuku.
